_maps/~monkestation/templates/hives.dmm

```diff
@@ -3,7 +3,9 @@
 /turf/closed/indestructible/hive,
```

Comment:
get rid of the changes to the map file - they'll be anchored as you have that set as the default for `/obj/structure/hive_exit`